.github/workflows/nodejs.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        node-version: [14.x, 16.x]
+        node-version: [16.x, 18.x]
```

Comment:
We should be able to safely drop Node.js 14 here (confirmed with Alena that VSCode is on 16.x).